eslint.config.js

```diff
@@ -0,0 +1,50 @@
+import baselinePlugin from 'eslint-plugin-baseline-js';
+import yalhPlugin from 'eslint-plugin-yet-another-license-header';
+import tseslint from 'typescript-eslint';
+
+const defaultLicense = `
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+`;
+
+const licensePattern =
+  /^\/\*\n \* Copyright The OpenTelemetry Authors(?:, [^\n]+)*\n(?: \* Copyright [^\n]+\n)* \*\n \* Licensed under the Apache License, Version 2.0 \(the "License"\);\n \* you may not use this file except in compliance with the License.\n \* You may obtain a copy of the License at\n \*\n \* {6}https:\/\/www.apache.org\/licenses\/LICENSE-2.0\n \*\n \* Unless required by applicable law or agreed to in writing, software\n \* distributed under the License is distributed on an "AS IS" BASIS,\n \* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n \* See the License for the specific language governing permissions and\n \* limitations under the License.\n \*\/$/;
+
+export default [
+  {
+    files: ['packages/*/src/**/*.{js,ts,mjs}'],
+    languageOptions: {
+      parser: tseslint.parser,
+    },
+    plugins: {
+      'baseline-js': baselinePlugin,
```

Comment:
Is this effectively duplicating biome linting? I understand they aren't identical.

Reply:
No, it's not. This is only running the baseline plugin and the header plugin which are not available in the biome plugin format. Both features are essential so while running biome and eslint is not ideal, it's still a major speed up from running eslint recommended rules along with all the necessary plugins.